azure-pipelines.yml

```diff
@@ -22,7 +22,7 @@ variables:
   (FullyQualifiedName!=Microsoft.Spark.E2ETest.UdfTests.UdfSimpleTypesTests.TestUdfWithReturnAsTimestampType)&\
   (FullyQualifiedName!=Microsoft.Spark.E2ETest.UdfTests.UdfSimpleTypesTests.TestUdfWithTimestampType)&\
   (FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.SparkSessionTests.TestCreateDataFrameWithTimestamp)"
-  LatestDotnetWorkerDir: '$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Worker\netcoreapp3.1\win-x64'
+  CurrentDotnetWorkerDir: '$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries\Microsoft.Spark.Worker\netcoreapp3.1\win-x64'
```

Comment:
I see "$(Build.ArtifactStagingDirectory)/Microsoft.Spark.Binaries" in many places. Can you create a variable for this?

Reply:
Do you mean to create two variables one with `/`, another with `\`?

Reply:
Can't they be used interchangeably?

Reply:
Added.